purchase_order_weight_volume/__manifest__.py

```diff
@@ -1,10 +1,13 @@
+# © 2023 ooops404
```

Comment:
pls replace the copyright symbol w/ "Copyright"

Reply:
ok